packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -183,7 +177,9 @@ const defaultCodeText = '# This environment contains the variable "model" \n# wh
 const codeText = ref(defaultCodeText);
 const llmQuery = ref('');
 const llmThoughts = ref<any[]>([]);
-const isSaveForReuseDisabled = ref(false);
+
+const isDraft = ref(false);
+let saveModel = false;
```

Comment:
I don't understand what this  variable saveModel is used for? can you provide some explanation?